app/helpers/task_helper.rb

```diff
@@ -20,19 +20,21 @@
 
 module TaskHelper
 
-  def format_reschedule task_id
-    html = '-'
-    icon = raw content_tag(:i, '', class: "glyphicon glyphicon-calendar")
-    html = raw content_tag(:a, raw(icon), class: 'task-reschedule', href: 'javascript:void(0)', task_id: task_id)
+  def format_reschedule(task_id)
```

Comment:
For all of these helpers, can you add `task` in the method names? Helpers can be weird because they don't scope to just one helper file, so it can be a problem if you end up with two methods in two different helpers that have the same name. I like to start the method with the object name